CHANGELOG.rst

```diff
@@ -1,5 +1,62 @@
 .. _changelog:
 
+0.6.3
+-----
+
+New feature
+~~~~~~~~~~~~
+
+⭐️ Project: commit list is now displayed
+
+Notable improvements
+~~~~~~~~~~~~~~~~~~~~
+
+🚄 Datasets: dataset creation and import unified in the UI
+
+🚄 Environments: auto-saved branches are filtered per username
```

Comment:
I still tried to sell it as a feature :grin: 
I think it makes sense to leave it like that since in both Release Notes and commit messages it's treated as a feature.
`feat(environments): filter autosaved branches per username`

Reply:
I was just going to comment the same about the release notes and the commit message :smile: